.travis.yml

```diff
@@ -11,10 +11,19 @@ before_install:
 install:
     - eval "$(gimme 1.11.4)"
 script:
-    -   cd go-controller;
-    -    make;
-    -    make windows;
-    -    make gofmt;
-    -    make install.tools;
-    -    make lint;
-    -    make check;
+    - pushd go-controller
+    -    make
+    -    make windows
+    -    make gofmt
+    -    make install.tools
+    -    make lint
+    -    make check
+    - popd
+    - |
```

Comment:
I have no clue about travis syntax (this isn't actually creating a pipeline, right?) but the check makes sense

Reply:
@danwinship I don't quite understand the part about "the check makes sense" here...

Reply:
I meant the part following this part looks like it does what it's supposed to